docs/interactions/Application_Commands.md

```diff
@@ -1,83 +1,130 @@
-# Slash Commands
+# Application Commands
 
-Slash Commands are the new, exciting way to build and interact with apps on Discord.
+Application commands are commands that an application can register to Discord. They generally have a name, description, and a set of options or arguments for a user to fill out. Unlike interactions, which can have dynamic data, application commands are guaranteed to be the same for any user that uses them.
```

Comment:
> Unlike interactions, which can have dynamic data, application commands are guaranteed to be the same for any user that uses them.

is slightly confusing wording imo -- you mean descriptions by "dynamic data," right? Not the response?

Reply:
Yeah this is poor wording, brain wasn't making good words. Will change.